### PR TITLE
Alt-Title Tweaks

### PR DESCRIPTION
**Pull request by SpadesNeil**
- ~~Cadet added to Assistant alt titles, because no assistant title exists for an aspiring Sec Officer.~~
- ~~Site Manager -> Colony Manager. I just thought it sounded better.~~
- ~~Overseer -> Colony Overseer. Again, just thought it sounded better.~~
- Added ~~"Colony Commander" and~~ "Site Supervisor" to Colony Director's alt titles.
- "Deputy Supervisor", "Deputy Director" added to HoP titles.
- "Administrative Assistant" added to Command Secretary.
- "Barkeeper" added to Bartender.
- "Gardener" is now an alt title. Botanist is the new default title. Gardener really isn't the correct title for what the role does by default, anyway.
- "Logistics Manager" added to Quartermaster.
- ~~"Caretaker",~~ "Maid" added to Janitor.
- "Cleric" added to Chaplain.
- "Head of Research" added to Research Director.
- Added "Cargo Courier" to Cargo Technician.
- ~~"Geologist",~~ "Physicist" added to Scientist.
- "Prosthesis " added to Roboticist.
- ~~"Biomechanical Engineer" -> "Biomechanical Technician" for Roboticist.~~
- "Commander" -> "Security Commander" because "Commander" was confusing for the HoS.
- ~~Added "Security Overseer" to HoS.~~
- Warden finally has alt titles of "Armorer", "Brig Manager"

I'm not really attached to any of these except I insist that "Commander" should be "Security Commander" because "Commander" alone is super confusing and also not correct terminology in the context of a research facility.

### CODE REVIEW

**Comment by Belsima** [comment]
Site Manager should be generic, given the upcoming southern cross.

**Comment by SpadesNeil** [comment]
A colony is a colony unless you're making another space station.

**Comment by Belsima** [comment]
The rest I support whole-heartedly.

**Comment by Anewbe** [comment]
It's another station. Get the full list in the opening post, please.

**Comment by SpadesNeil** [comment]
List posted. I originally had it all written but Github client ate my text.

**Comment by Belsima** [comment]
Mechatronic engineer back pls

My baby

**Comment by SpadesNeil** [comment]
:U Fine

**Comment by Anewbe** [comment]
These are my thoughts, before I go get Staff Input.

- Cadet will not be merged as an Assistant alt-title, this is non-negotiable. Maybe as another Sec officer title, but Cadet turns into Sec Tide.
- Site Manager is better, the aspecificity means we don't have to change it again when we go to the Southern Cross, or elsewhere.
- Same as above, Re: Overseer.
- Site Commander no, Site Supervisor maybe. Hell, I'd argue for that to be the default title.
- Logistics Manager, I don't know. Seems a bit grandiose for a glorified box pusher. Maybe as a HoP title instead?
- Caretaker implies to me a certain medical profession, not a cleaning lady.
- What Geology or Physics is really done on station?
- I'm fine with replacing the Engineer part of the Roboticist titles with Technician. Merge Android Techn and Cyborg Tech into one "Prosthesis Technician" title, those're frequently the same thing in our lore.
- Meant to add the Security Commander part myself.
- No Security Overseer, we don't need more confusion when people start using titles over radio.

**Comment by Atermonera** [comment]
"Colony" -manager, -overseer, etc., arent used because the director / w/e title they go by are just directors for the work level the current station is on, rather than the colony as a whole. Geologist will make a bit more sense as a xenoarch alt title when we move to the SC, but it doesn't make much sense on an asteroid colony outside academia, which is more adequately represented by the professor alt title we (iirc) have for librarian. Otw I share Anewbe's sentiments

**Comment by SpadesNeil** [comment]
"Director" was originally chosen because that's what NASA literally said was the correct title. :U I know because I personally nagged about it.

Site Supervisor should absolutely be an alt title though. I'd be okay with even replacing Director with Supervisor at some point, so as to lessen confusion with Research Director. Alternatively, make Research Director the Research Supervisor. Either would be fine. I think the latter makes more sense. Even 'Head of Research' sounds fine imo.

But that's all stuff for another PR. I'll make adjustments to this one for now.

**Comment by Anewbe** [comment]
Director gets confusing if there's a Research Director, and that's happened to me more than once.

File: code/game/jobs/job/civilian.dm [hunk 1]
```diff
@@ -145,6 +145,7 @@
 	idtype = /obj/item/weapon/card/id/cargo
 	access = list(access_maint_tunnels, access_mailsorting, access_cargo, access_cargo_bot, access_mining, access_mining_station)
 	minimal_access = list(access_maint_tunnels, access_cargo, access_cargo_bot, access_mailsorting)
+	alt_titles = list("Cargo Courier")
```
Comment (Anewbe) [inline code/game/jobs/job/civilian.dm]: You missed this one in your list, or I did.

File: code/game/jobs/job/science.dm [hunk 2]
```diff
@@ -57,8 +57,7 @@
 	economic_modifier = 7
 	access = list(access_robotics, access_tox, access_tox_storage, access_research, access_xenobiology, access_xenoarch)
 	minimal_access = list(access_tox, access_tox_storage, access_research, access_xenoarch)
-	alt_titles = list("Xenoarcheologist", "Anomalist", "Phoron Researcher")
-
+	alt_titles = list("Xenoarcheologist", "Anomalist", "Phoron Researcher", "Geologist", "Physicist")
```
Comment (Anewbe) [inline code/game/jobs/job/science.dm]: Still don't see exactly what those last two are for.

**Comment by SpadesNeil** [comment]
Want me to just change it now? I don't see why not.

I vote Research Director -> Research Supervisor, only because NASA said "Director" is the correct term for being in charge of the _whole facility_, and I will be annoyed if I went through the trouble of bugging NASA for nothing. :U

File: code/game/jobs/job/science.dm (hunk 2 above)
Comment (SpadesNeil) [reply code/game/jobs/job/science.dm]: For fun. It encourages science RP merely by the titles existing.

**Comment by SpadesNeil** [comment]
I insist on keeping Physicist for reasons of fun. It's a cool title that makes sense in the context of the station because we do have people who invent things that dick with the laws of physics, namely Bluespace stuff.

File: code/game/jobs/job/civilian.dm (hunk 1 above)
Comment (SpadesNeil) [reply code/game/jobs/job/civilian.dm]: I must have missed it in my list, lemmie mark it.

**Comment by Anewbe** [comment]
I don't really care what NASA says about management, I like them for the physics and shit, and that title's been a pain in my side since it was added.

**Comment by SpadesNeil** [comment]
Honestly Supervisor is totally fine.

Research Director or Colony Director can be changed to Supervisor for all I care. I have a _slight_ preference that we change Research Director to Research Supervisor, though.

**Comment by PrismaticGynoid** [comment]
Please don't touch the roboticist alt-titles. Biomechanical engineering is an actual thing, and they already deal with prosthetics and artificial organs.

**Comment by Wickedtemp** [comment]
I support the addition of "Chief Mlemical Officer" for an alternative title to CMO. 

Also, yeah, 'caretaker' for custodial staff is a little eh, it implies like...an Orderly or a social worker.

**Comment by SpadesNeil** [comment]
Forgot to cross out caretaker. It's gone now.

**Comment by Mechoid** [comment]
'Prostheticist' would likely be the correct name, if it ends up being kept. 'Prosthesis' is a false/replacement limb.

**Comment by Cerebulon** [comment]
Most of these seem like nothing more than synonyms without any significant difference in role and/or 'rank' which is in my mind the point of alt-titles. Others are overly specific (Maid) or oddly military (Cadet) or oddly archaic (Cleric). 

I do like:
"Site Supervisor" because yeah, it's generic to fit any kind of station or facility and I feel a TSC like NanoTrasen would want a standard title for that sort of thing.
"Deputy Director/Supervisor" because that's essentially the role they take.
"Gardener" and "Botanist" being switched as defaults.
"Commander" -> "Security Commander" makes a lot of sense to me.

"Prosthesis(ist)" is already covered by Biomechanical Engineer, that's what the title is supposed to cover anyway. Definitely on the side of not calling them technicians as that implies a lower level of skill than the people building entire robotic-things in a lab of sorts.

**Comment by SpadesNeil** [comment]
"Prostheticist" I kinda like better anyway. Can we get a third opinion before I decide to change it or not?

**Comment by Atermonera** [comment]
Technician implies technical school, which, while it does go into skilled trades, usually doesn't go to the levels of detail covered in university, where engineers come from. BiomechE's also do medical stuff, like installing and maintaining prosthetics that are actively installed. Doctors of any sort don't go to trade school, they go to medical school. While that may seem similar, Med school requires a pre-existing Uni degree or some equivalent.
+1 BiomechE

**Comment by Neerti** [comment]
Out of all the proposed alt title modifications, I'm only really for the security commander one (less confusing), and the gardener one (since it doesn't really change anything).

The rest of these I'm not in favor of, since they seem to go against my general idea of what an alt title is, of which I will explain below.  Warning: This is really long.

Alt titles are permanently tied to the job they're tied to, or they would just be new jobs in their own right, and so they're generally tightly bound to their original job.  As such, I can observe four different types of alt titles.

The first type is just a specialization of the various roles the base job is generally able to do.  There are actually two types that I can see within the specialization-based alt titles, which I will call 'hard spec' and 'soft spec'.  The difference between hard and soft is purely cultural to the server and isn't mechanically enforced, however 'hard spec' roles generally are expected to do their specialization above their base job's duties, where as 'soft spec' is generally more of the same from the base job, and might be indicative of a weak alt title or the game being too demanding on a job.  Hard specs aren't always 100% good and might be more worthy as their own jobs, but generally they can justify their existance.

Medical and Research alt titles generally contains hard specialization alt titles, with roles such as Surgeon, Virologist, Xenobiologist, Xenoflorist, Phoron Researcher, and such.  A lot of hard spec'd alt titles actually share a common base job, but culturally they are expected to remain separated, as it would be a bit odd to see a Virologist fixing someone's bones in surgery.  Sometimes in lop pop rounds, that might be necessary, however the dissonance partly comes from this expectation of hard specialization, and a case could be made for seperating these kinds of alt titles into their own true jobs (Virologist was actually its own slot before Polaris was a thing), however splitting or merging jobs tends to be very controversial.

'Soft spec', on the other hand, has a much smaller divide for the base job, and is generally more for flavor.  Engineering generally has these, with alt titles such as Electrician and Engine Technician, who have their alleged specializations (wires presumably, and the engine respectively), but in practice tend to play like most engineers and needing to do tasks such as starting the engine, fixing breaches, repairing computers, replacing cut wires, and such, due to the base job (Station Engineer) needing to fix everything that isn't pipe related, and generally they need to go do Atmos' job too if there isn't one present.

Soft specializations result from the needs of the game overpowering the idea of specialization for a specific title.  Imagine if there was an Electrician, who for whatever reason ICly can't start the engine.  They are the only engineer.  Barring some outside event like a borg doing it for them, they have two choices.  Consign the round to no power or break their specialization.  The pressure to break their spec is intense, as the entire round depends on power being available.  Low pop medical can suffer similar pressures.

Soft specs failing at actually encouraging specialization is not a fault of the player, but the game forcing it, however fixing that is generally very challenging, or sometimes impossible due to circumstances beyond our control (low-pop generally).  Adding a load of new alt titles such as Cargo Courier would just be adding more soft specs, which I'd rather not have, as they would play just like their base title and would, in my view, be rather pointless.  A lot of the alt Head of Staff titles feel pointless to me (Overseer, Crew Resources Director, Chief of Security, etc) as well since they play exactly like the base job, and I don't want to add even more useless titles.

Now, there is another type for some kinds of alt titles, that being allowing more types of characters to be able to play the base job, to a degree.  Generally this takes the form of having a 'less skilled' variant of the base job, for example Junior Officer, Nurse, Maintenance Technician, Cook.  These titles are meaningful because they let more kinds of characters play inside departments who otherwise would probably not be able to qualify for the base job (Doctor, Engineer, Chef, etc).  These alt titles are actually good for the game and not just a one line difference to the base job, and this justifies their existence.

The last type of alt title are those that are added for the sake of having more alt titles.  This PR feels like its trying to fill this category of alt titles, and I'm not in favor.  Alt titles like Professor, Prospector, Historian, Custodian, and such are 99% of the time unused or play exactly like their base job.  A lot of the alt titles proposed in this PR fit this group, such as Barkeeper, Cleric, Administrative Assistant, Physicist, etc, seem like hollow clones of their base job or have no actual means to do what the alt title implies what they should do, and as such I really don't want to add any more of those to the game, as those titles cannot justify their existence to me beyond 'I want my rank on the manifest to be slightly different'.

As such, I don't want most of the alt titles proposed here, and in the future I will likely voice my objections to adding more if they are just being added for the sake of doing so.

**Comment by Anewbe** [comment]
@SpadesNeil Any intention of adjusting this?

**Comment by SpadesNeil** [comment]
I thought I did. Which ones do you guys want adjusted/deleted? Big block of text is thorough but it makes it easy for me to miss stuff when I am going through the todo list.

**Comment by Anewbe** [comment]
I'm going to say that, for the time being, follow what Neerti has to say, because I'm in agreement with his argument.

**Comment by SpadesNeil** [comment]
Sooooo delete everything except Security Commander and the change to Gardener?

**Comment by Anewbe** [comment]
Pretty much.

**Comment by SpadesNeil** [comment]
Might as well make a new PR, then, since I wanna make one more sensible edit to Research Director to avoid further confusion. More on that shortly.

**Pull request closed without merging**